### ISSUE TITLE
Add support for "inputmode"

### ISSUE THREAD

**Issue by mpschaeuble** [udos86/ng-dynamic-forms#866]
## I'm submitting a

<pre><code>
[ ] Bug / Regression
[X] Feature Request / Proposal
[ ] Question
</code></pre>

## I'm using

<pre><code>
NG Dynamic Forms Version: `X.Y.Z`

[ ] Basic UI
[ ] Bootstrap UI  
[ ] Foundation UI
[ ] Ionic UI
[ ] Kendo UI
[x] Material  
[ ] NG Bootstrap
[ ] Prime NG
</code></pre>

## Description
It would be nice to add support for setting the "inputmode" attribute on a form field (see also https://html.spec.whatwg.org/multipage/interaction.html#input-modalities%3A-the-inputmode-attribute)

**Comment by udos86** [comment]
@mpschaeuble Hi!

Nice proposal but the overall browser support seems to be [very poor](https://caniuse.com/#feat=input-inputmode). 

Are there any know side effects?

**Comment by mpschaeuble** [comment]
I agree, browser support is limited. But in my opinion, this feature is most likely relevant on mobile devices and Android supports it (which is the most common OS for mobile devices). So a large user group should be able to make use of that feature.

**Comment by Karamuto** [comment]
@mpschaeuble ,
I agree this is actually a property for touch related mediums and these are used 80-90% by the android browser or chrome.

All in all it wouldn't hurt having this attribute as browser which don't know it just won't use it.

Would you make an pull request featuring this? It would be needed in all libraries expect ionic ( and kendo? )
